### DISCUSSION TITLE
Whats the best AI 1080-4K upsize out there now for stax

### DISCUSSION THREAD

**Discussion by chowderhead** [staxrip/staxrip#1543] [General]
I was told be a friend that stax 1080-4K upsize using NVVFX super upsize module  just made 4K that still looked like 1080.  Rather than stumble around I wondered what anyone else has experienced.  It a shhame becuz the BVVFX downsize super 4K-1080 does a fantastic job and ive received many compliments. vut not on the reverse.  Thanks.

**Comment by Dendraspis** [comment]
I can't really tell because I don't use `NVEncC` and I don't resize my videos, but I would suggest also trying the other algorithms `NVEncC` offers like `libplacebo` and `ngx-vsr`. Did you also play with the parameters of the algos such as *strength*?

Otherwise you can try filters for AviSynth/VapourSynth like `ResampleHQ`.

I hope to get AI based resizers included and running next year...
